paddlets/datasets/splitter.py

```diff
@@ -479,9 +479,9 @@ class SlideWindowSplitter(SplitterBase):
     def __init__(self,
                  train_size: int,
                  test_size: int,
-                 step_size: int = None,
-                 skip_size: int = 0,
-                 verbose: bool = True) -> None:
+                 step_size: Optional[int] = None,
+                 skip_size: Optional[int] = 0,
+                 verbose: Optional[bool] = True) -> None:
```

Comment:
Line484 verbose should Not be Optional, refers to  https://docs.python.org/3/library/typing.html#typing.Optional